scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -1,13 +1,15 @@
 # MXNet Java Sample Project
 This is an project created to use Maven-published Scala/Java package with two Java examples.
 ## Setup
-You can use the `Makefile` to make the Java package. Simply do the following:
-```Bash
-make javademo
+User are required to use maven to build the package, by simply running
```

Comment:
```suggestion
You are required to use Maven to build the package with the following commands:
```
However, isn't this not true? You can run the sample project by building for source, can you not?